### PR TITLE
Reindex case views

### PR DESCRIPTION
**Pull request by dannyroberts**
Depends on:
- [x] https://github.com/dimagi/commcare-hq/pull/7174
- [x] https://github.com/dimagi/commcare-hq/pull/7169

### CODE REVIEW

**Comment by dimagimon** [comment]
Could not review pull request. It may be too large, or contain no reviewable changes.

**Comment by dannyroberts** [comment]
I'll probably make a few more changes deleting or changing views in this design doc; I'll add them here and eventually merge them all together for one big reindex

**Comment by czue** [comment]
whee

File: corehq/apps/casegroups/models.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    """
+        This is a group of CommCareCases. Useful for managing cases in larger projects.
+    """
+    name = StringProperty()
+    domain = StringProperty()
+    cases = ListProperty()
+    timezone = StringProperty()
+
+    def get_time_zone(self):
+        # Necessary for the CommCareCaseGroup to interact with CommConnect, as if using the CommCareMobileContactMixin
```
Comment (dimagimon) [inline corehq/apps/casegroups/models.py]: E501 line too long (118 > 115 characters)

File: corehq/ex-submodules/casexml/apps/case/models.py [hunk 2]
```diff
@@ -1025,3 +954,7 @@ def _type_sort(action_type):
     Consistent ordering for action types
     """
     return const.CASE_ACTIONS.index(action_type)
+
+
+# import signals
+import casexml.apps.case.signals
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/models.py]: F401 'casexml' imported but unused

File: corehq/ex-submodules/casexml/apps/case/_design/views/open_cases/map.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+function(doc) {
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/_design/views/open_cases/map.js]: Missing name in function declaration.

**Comment by benrudolph** [comment]
:white_check_mark: after tests

**Comment by dannyroberts** [comment]
@czue FYI self-merging and starting preindex. Removed 2 'case' views, simplified another one to emit 1/4 the things, and moved another to a separate app.

**Comment by czue** [comment]
awesome!

**Pull request merged**